OrbitQt/main.cpp

```diff
@@ -404,6 +497,11 @@ int main(int argc, char* argv[]) {
       return -1;
     }
 
+    if (absl::GetFlag(FLAGS_enable_ui_beta)) {
+      RunBetaUiInstance(&app, deployment_configuration, &context.value());
+      return 0;
+    }
+
     while (true) {
```

Comment:
I think we should move everything below into "RunUiInstance", so the flow of this method is clearer ("if ui_beta then RunBetaUiInstance else RunUiInstance").

Reply:
Well... since that was a bit annoying and is only a temporary solution, I at least scoped it into a "else" part for clarity now. @beckerhe PTAL

Reply:
I also thought about this and came to the same conclusion: It would be nice to have a single function and not duplicate code, but it's also hard. So I'm fine with leaving that. Like you said, it's very temporary.